api/app/Models/Taxonomy.php

```diff
@@ -6,9 +6,10 @@
 class Taxonomy extends Model{
 
 	protected $table = 'taxa';
+	public $timestamps = false;
 	protected $primaryKey = 'tid';
-	protected $hidden = [ 'sciName', 'phyloSortSequence', 'nomenclaturalStatus', 'nomenclaturalCode', 'statusNotes', 'hybrid', 'pivot', 'modifiedUid', 'modifiedTimeStamp', 'initialTimeStamp', 'InitialTimeStamp' ];
-	protected $fillable = [  ];
+	protected $hidden = [ 'phyloSortSequence', 'nomenclaturalStatus', 'nomenclaturalCode', 'statusNotes', 'hybrid', 'pivot', 'modifiedUid', 'modifiedTimeStamp', 'initialTimeStamp', 'InitialTimeStamp' ]; // @TODO I'm  not sure why sciName was hidden, but I don't think that it can be if we need to POST sciName with an endpoint
```

Comment:
TODO I just realized that I/we need to test whether this broke taxonomy GET endpoints.

Reply:
I still have to test it. I think it can't be hidden if we want it to work with the POST endpoint. I'll research and try to implement a workaround.